text_generation/causal_lm/cpp/continuous_batching/library/src/paged_attention_transformations.cpp

```diff
@@ -16,5 +19,32 @@ void apply_paged_attention_transformations(std::shared_ptr<ov::Model> model, con
         parameters[2 + 2 * decoder_layer_id]->set_partial_shape(device_config.get_key_cache_shape());
         parameters[2 + 2 * decoder_layer_id + 1]->set_partial_shape(device_config.get_value_cache_shape());
     }
+
+    if (scheduler_config.dynamic_split_fuse) {
```

Comment:
maybe we can introduce new parameter in scheduler_config to denote optimization of different input lengths? 
it can be useful for vLLM case as well, when on prompt stage we can avoid padding and set `subsequence_lens` to real lengths of prompts

Reply:
To avoid padding, may we use `_forward_generate` for prompts?